packages/vscode-extension/src/dependency/DependencyChecker.ts

```diff
@@ -123,21 +123,32 @@ export class DependencyChecker implements Disposable {
   }
 
   public async checkCocoaPodsInstalled() {
-    const installed = await checkIfCLIInstalled("pod --version", {
-      env: { ...process.env, LANG: "en_US.UTF-8" },
+    const env = { ...process.env, LANG: "en_US.UTF-8" };
+    let isCocoaPodsInstalled = await checkIfCLIInstalled("pod --version", {
+      env,
     });
+
+    // If `pod --version` fails, try `bundle exec pod --version` as a second attempt
+    if (!isCocoaPodsInstalled && workspace.workspaceFolders?.length) {
+      isCocoaPodsInstalled = await checkIfCLIInstalled("bundle exec pod --version", {
+        env,
+        // Run the command in the current project directory
+        cwd: workspace.workspaceFolders?.[0].uri.fsPath,
```

Comment:
Can we safely assume that first workspace directory is always an app directory?

Reply:
Not always, it largely depends on the users' setup but this is a fair try IMHO.
We could attempt to look for the correct folder within the `fsPath`, though.